NATS.Jwt/Models/NatsClientInformation.cs

```diff
@@ -23,7 +23,7 @@ public record NatsClientInformation
     /// </summary>
     [JsonPropertyName("id")]
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
-    public string Id { get; set; }
+    public long Id { get; set; }
```

Comment:
are you sure about this one? it looks like it needs to represent base32 strings of nkeys public keys

Reply:
Actually, check the [output here](https://gist.github.com/rickdotnet/2c556cb218b1fa2493da4a38c66fad05). The client id is `31` as opposed to `"31"`

```
{
  "aud": "nats-authorization-request",
  "exp": 1723166342,
  "jti": "2KW2GUTFIO4VQMK5QUB5IU7FE2F365MTXYEKKGXCZGMIELMYGEOQ",
  "iat": 1723166340,
  "iss": "NAL2F6RPUKRNRVTXT6XDHGZWUWD54GWZZFIJ5KU5OHEURSVWS7T4GBPZ",
  "sub": "AACILD5SO3CCBAT5JJVOTWLVBLABXLPP2Q76SI5KSUYFV7STPTALPAG2",
  "nats": {
    "server_id": {
      "name": "NAL2F6RPUKRNRVTXT6XDHGZWUWD54GWZZFIJ5KU5OHEURSVWS7T4GBPZ",
      "host": "0.0.0.0",
      "id": "NAL2F6RPUKRNRVTXT6XDHGZWUWD54GWZZFIJ5KU5OHEURSVWS7T4GBPZ",
      "version": "2.10.18"
    },
    "user_nkey": "UCO3JSTTM3ZDZ6QPTED2SM472PHK264ZHWGPU5I4OQD4AWLTWYF5BLIQ",
    "client_info": {
      "host": "::1",
      "id": 31,
      "user": "bob",
      "name": "NATS CLI Version 0.1.4",
      "kind": "Client",
      "type": "nats"
    },
    "connect_opts": {
      "user": "bob",
      "pass": "bob",
      "name": "NATS CLI Version 0.1.4",
      "lang": "go",
      "version": "1.33.1",
      "protocol": 1
    },
    "type": "authorization_request",
    "version": 2
  }
}
```

Reply:
> it looks like it needs to represent base32 strings of nkeys public keys

Where do you see this? I need to update my notes with a good references of these types.

Reply:
my mistake thanks for the detail. I copied from here: https://github.com/nats-io/jwt/blob/7b33986cffa3bd7df2277d853249c385edf9315f/v2/authorization_claims.go#L38